odoo/addons/base/models/res_config.py

```diff
@@ -452,11 +448,19 @@ def _get_classified_fields(self):
                 }
         """
         IrModule = self.env['ir.module.module']
+        IrModelData = self.env['ir.model.data']
         Groups = self.env['res.groups']
-        ref = self.env.ref
+
+        def ref(xml_id):
+            model, id = IrModelData._xmlid_to_res_model_res_id(xml_id)
```

Comment:
```suggestion
            res_model, res_id = IrModelData._xmlid_to_res_model_res_id(xml_id)
```
this prevents the runbot to pretend you style sucks :stuck_out_tongue_winking_eye:

Reply:
ci/style didn't exist when I first made this branch :D